docker/generate_dockerfiles.sh

```diff
@@ -52,77 +52,70 @@ do
   esac
 done
 
-
-# neurodocker version 0.3.1-22-gb0ee069
-NEURODOCKER_IMAGE="kaczmarj/neurodocker@sha256:f15ca90803f4b89acfca55cd1c7269bf2ec2dfd95b3de1118b08afa34b87d9d1"
-
-# neurodebian:stretch-non-free pulled on November 3, 2017
-BASE_IMAGE="neurodebian@sha256:7590552afd0e7a481a33314724ae27f76ccedd05ffd7ac06ec38638872427b9b"
+# neurodocker version 0.4.1-22-g7c44e01
+NEURODOCKER_IMAGE="kaczmarj/neurodocker:master@sha256:858632a7533cac100f70932749b4cfc77fc40f667f41fca208f406215cff8a27"
+# neurodebian:stretch-non-free pulled on September 19, 2018
+BASE_IMAGE="neurodebian:stretch-non-free@sha256:7cd978427d7ad215834fee221d0536ed7825b3cddebc481eba2d792dfc2f7332"
```

Comment:
@kaczmarj What if we revert the `BASE_IMAGE`? If that helps, I think that would be decent evidence that it's a library version issue.

Reply:
@effigies - i don't think that will help. the `apt-get update` calls will update to the latest neurodebian stretch sources. i tried using neurodebian snapshots from september 17, 2017 and from january 1, 2017, but both ran into the same `flameo` error. i'm going to try to use the nipype base image on dockerhub with the current nipype source code now.

Reply:
@effigies - i used nipype's base image on dockerhub (i.e., `nipype/nipype:base`) and still got the `flameo` error. is it possible that nipype's code is causing this? could one of the inputs to `flameo` be corrupted / malformed?

Reply:
But it works on `master`, still, with the same inputs. There's something new in the build.

Reply:
@kaczmarj Is there any chance of using a Debian snapshot from earlier, as well? If it's a libc update or similar that's causing the issue, that wouldn't be resolved by using a neurodebian snapshot.

Reply:
true, i'll try that now. i'm experimenting with updating packages in the current nipype py36 docker image and see if that will reproduce the error.